packages/webapp/pages/account/profile.tsx

```diff
@@ -63,6 +63,7 @@ const AccountProfilePage = (): ReactElement => {
       update: true,
     }),
   );
+  const [updatedUser, setUpdatedUser] = useState(user);
```

Comment:
only thing that comes to mind is that `user` here could be `undefined` on initial render and then we get no values in `updatedUser` but it should not happen since it comes from boot? We need forms lib!

Reply:
I'm open to just deleting this whole PR as Lee's fix is enough for now.